src/query/frontend/ast/cypher_main_visitor.cpp

```diff
@@ -3188,4 +3188,19 @@ antlrcpp::Any CypherMainVisitor::visitAlterEnumUpdateValueQuery(MemgraphCypher::
   return alter_enum_query;
 }
 
+antlrcpp::Any CypherMainVisitor::visitAlterEnumRemoveValueQuery(MemgraphCypher::AlterEnumRemoveValueQueryContext *ctx) {
+  auto *alter_enum_query = storage_->Create<AlterEnumRemoveValueQuery>();
+  alter_enum_query->enum_name_ = std::any_cast<std::string>(ctx->enumName()->symbolicName()->accept(this));
```

Comment:
If you are using `symbolicName` a name like `1enum` will be stripped and generate `0 enum`.
Is this handled correctly?